projects/cashmere/src/lib/button-toggle/button-toggle.component.ts

```diff
@@ -47,7 +47,7 @@ export class ButtonToggleComponent {
             this._hostClass += ' hc-toggle-disabled';
         }
         if ( !this._disabled && this._hostClass.includes( 'hc-toggle-disabled' )) {
-            this._hostClass.replace( ' hc-toggle-disabled', '' );
+            this._hostClass = this._hostClass.replace( ' hc-toggle-disabled', '' );
```

Comment:
is the leading space on `' hc-toggle-disabled'` a typo or intentional?

Reply:
Yeah that one is actually intentional because `hostClass` is a list of classes on the element - without that leading space, it'll leave a whitespace character at the end of the class list which Angular doesn't like